ee-9/common/pom.xml

```diff
@@ -802,6 +812,17 @@
             </exclusions>
         </dependency>
 
+       <dependency>
+            <groupId>${ee.maven.groupId}</groupId>
+            <artifactId>jipijapa-hibernate6</artifactId>
+           <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
```

Comment:
Please add an entry for this to https://github.com/wildfly/wildfly/blob/main/ee-9/common/src/main/resources/license/preview-feature-pack-common-licenses.xml

Reply:
Thanks Brian, entry added!